src/System.IO/tests/Stream/Stream.TestLeaveOpen.cs

```diff
@@ -22,37 +22,20 @@ public void StreamReaderTest()
             ms.Position = 0;
             Assert.True(ms.CanRead, "ERROR: Before testing, MemoryStream's CanRead property was false!  What?");
 
-            // Test leaveOpen with calling Dispose
+            // Test leaveOpen.
             StreamReader sr = new StreamReader(ms, System.Text.Encoding.UTF8, true, 1000, true);
             sr.Dispose();
-            Assert.True(ms.CanRead, "ERROR: After disposing a StreamReader with leaveOpen bool set, MemoryStream's CanRead property was false!");
```

Comment:
No lets avoid more #ifdefs but we should have an issue file for adding these test correctly.